services/supervisor/tests/supervisor/inc.rs

```diff
@@ -15,12 +15,12 @@
 use serde_derive::{Deserialize, Serialize};
```

Comment:
I wonder if we expose services like this from `testkit` we will avoid copy paste. Worth thinking.